app/models/student.rb

```diff
@@ -113,6 +113,32 @@ def eligible_for_trip(trip)
       has_consented_stim && has_signed_safety_test
   end
 
+  def has_forms_for_year
```

Comment:
This method doesn't quite know what its identity is. The question asked by this method is "does THIS student have all their forms for this year?" and the answer is an email and a bunch of strings.

The concept of a CSV is commingled with this method and you're returning strings formatted a particular way to satisfy that.

There's also no need to return a personal email. This method is being called on an instance of Student already.

Imagine you were asked to add a webpage to show this info or a JSON report in addition to the CSV report. How would your method change to accommodate that?

Reply:
This method is missing tests